app/models/bulkrax/csv_entry.rb

```diff
@@ -147,15 +148,17 @@ def prepare_export_data(datum)
     end
 
     def object_metadata(data, object_key)
+      # eval turns all of the keys into symbols
       data = data.map { |d| eval(d) }.flatten # rubocop:disable Security/Eval
       data.each_with_index do |obj, index|
-        next unless obj[object_key]
-        if obj[object_key].is_a?(Array)
-          obj[object_key].each_with_index do |_nested_item, nested_index|
-            self.parsed_metadata["#{key_for_export(object_key)}_#{index + 1}_#{nested_index + 1}"] = prepare_export_data(obj[object_key][nested_index])
+        object_symbol = object_key.to_sym
```

Comment:
initially I reassigned `object_key` to be a symbol. but that caused an issue when being passed into the `key_for_export` method. so instead of reconverting `object_key` into a string inside of `key_for_export`, I created the new variable `object_symbol`.

Reply:
I would have thought these needed to be strings all the time. are you sure they need to be symbols?

Reply:
in my testing I discovered that the eval version of `data` returned
```
[{:creator_family_name=>"Kornicki", :creator_given_name=>"Peter Francis", :creator_isni=>"110780723", :creator_name_type=>"Personal"}]
```

therefore, `obj[object_key]` was always blank because it was looking for a key with a data type of string, and not symbol.

Reply:
although....the specs _are_ failing for the opposite reason. they are expecting strings and not symbols. 🤔

Reply:
this was resolved by calling `with_in_different_access` on the obj hash inside of the `each`.